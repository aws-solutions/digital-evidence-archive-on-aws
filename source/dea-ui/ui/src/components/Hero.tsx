/*
 *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 *  SPDX-License-Identifier: Apache-2.0
 */

import { useSettings } from '../context/SettingsContext';
import styles from '../styles/Hero.module.scss';

function Hero(): JSX.Element {
  const { settings } = useSettings();

  return (
    <div className="custom-home__header">
      Hello {settings.name}!<button className={styles.primaryButton}>Go to dashboard</button>
    </div>
  );
}

export default Hero;
