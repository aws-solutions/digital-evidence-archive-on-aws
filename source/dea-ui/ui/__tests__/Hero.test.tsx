import { render } from '@testing-library/react';
import Hero from '@/components/Hero';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const { asFragment } = render(<Hero />);
    expect(asFragment()).toMatchSnapshot();
  });
});
