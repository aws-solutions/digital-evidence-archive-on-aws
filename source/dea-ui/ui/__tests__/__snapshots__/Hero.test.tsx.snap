// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<DocumentFragment>
  <div
    class="custom-home__header"
  >
    Hello Digital Evidence Archive on AWS!
    <button
      class="primaryButton"
    >
      Go to dashboard
    </button>
  </div>
</DocumentFragment>
`;
